src/cc_catalog_airflow/dags/finnish_museums_workflow.py

```diff
@@ -0,0 +1,31 @@
+"""
+This file configures the Apache Airflow DAG to ingest Finnish museums data.
+
+We do this by running `provider_api_scripts.finnish_museums.main`
+"""
+# airflow DAG (necessary for Airflow to find this file)
+from datetime import datetime, timedelta
+import logging
+
+from provider_api_scripts import finnish_museums
+from util.dag_factory import create_provider_api_workflow
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s:  %(message)s',
+    level=logging.INFO
+)
+
+logger = logging.getLogger(__name__)
+
+DAG_ID = 'finnish_museums_workflow'
+START_DATE = datetime(2020, 11, 23)
+DAGRUN_TIMEOUT = timedelta(hours=24)
```

Comment:
Please set the start date to 2020-09-01 (this is due to some eccentricities of how Airflow handles scheduling.

Reply:
I have made the changes